modules/timeplus/build.gradle

```diff
@@ -0,0 +1,10 @@
+description = "Testcontainers :: JDBC :: Timeplus"
+
+dependencies {
+    api project(':testcontainers')
+    api project(':jdbc')
+
+    testImplementation project(':jdbc-test')
+    testRuntimeOnly 'com.timeplus:proton-jdbc:0.6.0'
```

Comment:
`proton-jdbc` ?

Reply:
refer to https://github.com/testcontainers/testcontainers-java/blob/main/modules/clickhouse/build.gradle#L8